metadata-ingestion/src/datahub/entrypoints.py

```diff
@@ -58,6 +59,26 @@ def version() -> None:
     click.echo(f"Python version: {sys.version}")
 
 
+@datahub.command()
+def init() -> None:
+    """Configure which datahub instance to connect to"""
+    if os.path.isfile(DATAHUB_CONFIG_PATH):
+        click.confirm(f"{DATAHUB_CONFIG_PATH} already exists. Overwrite?", abort=True)
+
+    click.echo("Configure which datahub instance to connect to")
+    host = click.prompt(
+        "Enter your DataHub host", type=str, default="http://localhost:8080"
```

Comment:
Let's specify that this is GMS API server

Reply:
Some users might not know what GMS is, but I'll add a line in the docs to clarify this :)